packages/core/src/components/OverflowTooltip/OverflowTooltip.tsx

```diff
@@ -40,6 +40,10 @@ export const OverflowTooltip = (props: Props) => {
     setHover(truncated);
   };
 
+  if (!props.text) {
+    return null;
+  }
+
   return (
     <Tooltip
       title={props.title ?? props.text!}
```

Comment:
Think we should be able to remove the `!` from `props.text` now too, and then it's good to go for me!